index.html

```diff
@@ -5,50 +5,217 @@
+                            I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+                        </div>
+                    </section>
+                    <div class="contact">
+                        <h3 class="am-connect">LET'S CONNECT</h3>
+                    </div>
+                    <div class="connect-logo">
+                        <div class="logo">
+                            <ul class="amlogo-div">
+                                <li><a href=""><img src="./image/twiter.svg" alt=""></a></li>
+                                <li><a href=""><img src="./image/Vector.png" alt=""></a></li>
+                                <li><a href=""><img src="./image/M.svg" alt=""></a></li>
+                                <li><a href=""><img src="./image/Vector(3).svg" alt=""></a></li>
+                                <li><a href=""><img src="./image/Vector(4).svg" alt=""></a></li>
+                            </ul>
+                        </div>
+                    </div>
+                    <div class="see-project">
+                        <button type="button">Get my resume</button>
+                    </div>
+                        <div class="am-menu">
+                            <div class="languages">
+                                <div class="lang-menu">
+                                    <div class="lan-text">Languages</div>
+                                    <div class="lang-logo"><img src="./image/down_arrow.png" alt=""></div>
+                                </div>
+                                <div class="javascript">
+                                    <div class="ln-image"><img src="./image/javascript.png" alt=""></div>
+                                    <div class="ln-text">JavaScript</div>
+                                </div>
+                                <div class="html">
+                                    <div class="ln-image"><img src="./image/html.png" alt=""></div>
+                                    <div class="ln-text">HTML</div>
+                                </div>
+                                <div class="css">
+                                    <div class="ln-image"><img src="./image/css.png" alt=""></div>
+                                    <div class="ln-text">CSS</div>
+                                </div>
+                            </div>
+                            
+                            <div class="frame">
+                                <!-- <nav> -->
+                                    <div>Frameworks</div>
+                                    <div><img src="./image/right_arrow.png" alt=""></div>
+                                <!-- </nav> -->
+                            </div>
+                            <div class="skills">
+                                <!-- <nav> -->
```

Comment:
- [ ] [OPTIONAL] kindly remove all unused blocks from your code instead of commenting them out, this will make your code cleaner and easy to read